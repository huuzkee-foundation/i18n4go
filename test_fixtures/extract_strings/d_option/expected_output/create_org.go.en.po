# filename: d_option/input_files/org/create_org.go, offset: 793, line: 29, column: 16
msgid "create-org"
msgstr "create-org"

# filename: d_option/input_files/org/create_org.go, offset: 822, line: 30, column: 16
msgid "co"
msgstr "co"

# filename: d_option/input_files/org/create_org.go, offset: 1093, line: 38, column: 20
msgid "Incorrect Usage"
msgstr "Incorrect Usage"

# filename: d_option/input_files/org/create_org.go, offset: 1323, line: 52, column: 13
msgid "Creating org %s as %s..."
msgstr "Creating org %s as %s..."

# filename: d_option/input_files/org/create_org.go, offset: 1610, line: 60, column: 16
msgid "Org %s already exists"
msgstr "Org %s already exists"

# filename: d_option/input_files/org/create_org.go, offset: 1725, line: 68, column: 13
msgid "\nTIP: Use '%s' to target new org"
msgstr "\nTIP: Use '%s' to target new org"

# filename: d_option/input_files/org/create_org.go, offset: 843, line: 31, column: 16
msgid "Create an org"
msgstr "Create an org"

# filename: d_option/input_files/org/create_org.go, offset: 875, line: 32, column: 16
msgid "CF_NAME create-org ORG"
msgstr "CF_NAME create-org ORG"

# filename: d_option/input_files/org/create_org.go, offset: 1794, line: 68, column: 82
msgid " target -o "
msgstr " target -o "

