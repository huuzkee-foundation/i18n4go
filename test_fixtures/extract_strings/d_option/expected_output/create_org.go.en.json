[{"id":"Create an org","translation":"Create an org"},{"id":"CF_NAME create-org ORG","translation":"CF_NAME create-org ORG"},{"id":" target -o ","translation":" target -o "},{"id":"create-org","translation":"create-org"},{"id":"co","translation":"co"},{"id":"Incorrect Usage","translation":"Incorrect Usage"},{"id":"Creating org %s as %s...","translation":"Creating org %s as %s..."},{"id":"Org %s already exists","translation":"Org %s already exists"},{"id":"\nTIP: Use '%s' to target new org","translation":"\nTIP: Use '%s' to target new org"}]
